### PR TITLE
New Github Organization Application Installation Rule

### PR DESCRIPTION
**Pull request by andrea-youwakim**
### Background

<High level overview here>

### Changes

Added newly written rule to monitor activity related to installing a github organization application, fixed key ordering in pack file, referenced the rule ID in the github pack, and added global helper to pack.

### Testing

* Test Cases Included in Yml file.
* results linked below: (updated to include bracketed interpolated values in results)
Screen Shot 2022-12-13 at 1 14 40 PM

### CODE REVIEW

**Comment by andrea-youwakim** [comment]
@jpl5280 no problem- just uploaded one! test one is from an actual event pulled from the data explorer (with the p_values removed) and the second one is the one you've included in the ticket.

**Comment by jpl5280** [comment]
@andrea-youwakim - id suggest surrounding interpolated values with single-quotes or brackets

E.g. 

`Github user 'leetboy' in 'example-io' installed the following integration: Datadog CI`

or

`Github user [leetboy] in [example-io] installed the following integration: [Datadog CI]`

@edyesed @calkim-panther - do we have a best practice/standard for this?

**Comment by calkim-panther** [comment]
> 

We typically wrap interpolated values in brackets

File: rules/github_rules/github_organization_app_integration_installed.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Enabled: false
+Filename: github_organization_app_integration_installed.py
+Reference: https://docs.github.com/en/enterprise-server@3.4/developers/apps/managing-github-apps/installing-github-apps
+Runbook: Confirm that the app integration installation was a desired behavior.
+Severity: Low
+Tags:
+    - Application Installation
+    - Github
+Tests:
+    - ExpectedResult: true
```
Comment (edyesed) [inline rules/github_rules/github_organization_app_integration_installed.yml:13]: we should provide a `ExpectedResult: false` test case to demonstrate that there are cases where the alert will not be raised.

Comment (andrea-youwakim) [reply rules/github_rules/github_organization_app_integration_installed.yml:13]: ack; added this test case

**Comment by andrea-youwakim** [comment]
@jpl5280 updated the test results in the description to include bracketed interpolated values.

**Review by edyesed — APPROVED** [review]
adds new github audit based detection for a new integration being installed into a github organization.

Extends the github pack to  include new detection, and adds panther_base_helpers to the github pack in order to leverage `github_alert_context()` as the alert_context

**Pull request merged**